Lib/tkinter/test/test_tkinter/test_misc.py

```diff
@@ -156,6 +156,24 @@ def callback():
         with self.assertRaises(tkinter.TclError):
             root.tk.call('after', 'info', idle1)
 
+    def test_winfo_rgb(self):
+        root = self.root
+        rgb = root.winfo_rgb
+
+        # Color name.
+        self.assertEqual(rgb('red'), (65535, 0, 0))
```

Comment:
Add a test for a multiword color, e.g. `'dark slate blue'`.

Reply:
done (below)